### PR TITLE
news: add something about the upcoming ASLR change

### PR DESCRIPTION
**Pull request by lazka**
See https://github.com/msys2/MINGW-packages/pull/7859

### CODE REVIEW

File: docs/news.md [hunk 1]
```diff
@@ -4,6 +4,31 @@ summary: Important events happening.
...
+of it can be disabled/reverted via linker flags:
+
+* mingw64: `-Wl,--disable-dynamicbase,--disable-high-entropy-va,--disable-nxcompat,--default-image-base-low`
+* mingw32: `-Wl,--disable-dynamicbase,--disable-nxcompat`
+
+Note that this is only a workaround and some of these flags will not be
+available forever, so you should either fix your code or file a bug in case you
+suspect a toolchain issue.
+
+Thanks to the binutils developers for improving/fixing ASRL support and to
```
Comment (Biswa96) [inline docs/news.md:27]: Typo, ASLR.

File: docs/news.md [hunk 2]
```diff
@@ -4,6 +4,31 @@ summary: Important events happening.
...
+### 2021-01-31 - ASLR enabled by default
+
+**Backstory:** About 5 months ago we started backporting patches to our binutils
+2.35 to allow enabling ASLR support via various flags. We also enabled these
+flags in our build system, so any package in our repo that was updated in the
+last 5 months has ASLR support enabled. We've now updated to 2.36 which in
+theory has ASLR enabled by default but have reverted the defaults for now to
+make debugging easier in case of regressions.
+
+**Next steps:** In the coming days we will enable ASLR for everyone by default.
```
Comment (Biswa96) [inline docs/news.md:16]: That should be "for every packages by default". I was not built with ASLR 🤣

Comment (naveen521kk) [reply docs/news.md:16]: yeah yeah :laughing:

File: docs/news.md (hunk 1 above)
Comment (lazka) [reply docs/news.md]: thanks

File: docs/news.md (hunk 2 above)
Comment (lazka) [reply docs/news.md]: you're missing out ;)

**Comment by lazka** [comment]
I've added one more paragraph for "known issues"

**Review by naveen521kk — APPROVED** [review]

**Pull request merged**